subprojects/docs/src/docs/userguide/jvm/toolchains.adoc

```diff
@@ -15,31 +15,36 @@
 [[toolchains]]
 = Toolchains for JVM projects
 
-By default, Gradle uses the same Java version for running Gradle itself and building JVM projects.
+Working on multiple projects can require interacting with multiple versions of the Java language.
+Even within a single project different parts of the codebase may be fixed to a certain language level due to backward compatibility requirements.
+Because of this, different versions of the same tools (a toolchain) must be installed and managed on each machine that builds the project.
 
-This is not always desirable.
+A **Java toolchain** is a set of tools to build and run Java projects, which is usually provided by the environment via local JRE or JDK installations.
+Compile tasks may use `javac` as their compiler, test and exec tasks may use the `java` command while `javadoc` will be used to generate documentation.
+
+By default, Gradle uses the same Java toolchain for running Gradle itself and building JVM projects.
+However, this may not be always desirable.
```

Comment:
```suggestion
However, this may only sometimes be desirable.
```

Reply:
it is always best to choose positive terms. People don't want to know what "not" to do.